src/common/icons/Icons.tsx

```diff
@@ -446,3 +452,112 @@ export const EyeIcon = (): JSX.Element => (
 export const EyeSlashIcon = (): JSX.Element => (
   <IconContainer className="fa fa-eye-slash" />
 )
+
+const ZOOM_ICONS_DEFAULT_SIZE_IN_PX = 20
+const ZOOM_ICONS_LARGE_SCALE_FACTOR = 1.2
+export const ZoomInIcon = ({ large }: { large: boolean }): JSX.Element => {
+  const scale = large ? ZOOM_ICONS_LARGE_SCALE_FACTOR : 1
+  return (
+    <IconContainer
+      icon={zoomInIcon}
+      width={scale * ZOOM_ICONS_DEFAULT_SIZE_IN_PX}
+    />
+  )
+}
+export const ZoomOutIcon = ({
+  large = false
+}: {
+  large?: boolean
```

Comment:
is there a reason the "large" prop is optional only for zoom out icon? 😄

Reply:
hehe no, thanks, I'll fix!